src/templates/upload-dropdown-menu.html

```diff
@@ -48,8 +48,8 @@
   {% endif %}
 
     <!-- Show content of plain text -->
-    {% if ext matches '/txt$/i' %}
-      <a class='dropdown-item' data-action='show-plain-text' data-storage='{{ upload.storage }}' data-path='{{ upload.long_name }}' data-name='{{ upload.real_name }}'>
+    {% if ext matches '/(txt|md)$/i' %}
+      <a class='dropdown-item' data-action='show-plain-text' data-ext='{{ ext|raw }}' data-storage='{{ upload.storage }}' data-path='{{ upload.long_name }}' data-name='{{ upload.real_name }}'>
```

Comment:
is it necessary to have the `raw`?

Reply:
Probably not. `raw` is either `md` or `txt`. So no escaping anyways, it can be dropped.